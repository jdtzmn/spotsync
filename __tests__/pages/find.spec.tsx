import * as React from 'react'
import { shallow } from 'enzyme'
import Find from 'pages/Find'

describe('Find', () => {
  let wrapper
  beforeEach(() => {
    // Mock Math.random()
    const mockMath = Object.create(global.Math)
    mockMath.random = () => 0.000000
    global.Math = mockMath

    wrapper = shallow(<Find />)
  })

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
