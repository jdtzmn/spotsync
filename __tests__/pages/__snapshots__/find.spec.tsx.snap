// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Find should render correctly 1`] = `
<main>
  <div>
    <div>
      <h1>
        Find a Party
      </h1>
      <sub>
        Choose an option below
      </sub>
      <div>
        <LinkRoutes
          key="0"
          route="/code"
        >
          <button>
            <IosBarcodeOutline
              beat={false}
              color="#7A7A7A"
              fontSize="60px"
              rotate={false}
              shake={false}
              style={Object {}}
            />
            <h3>
              Use a code
            </h3>
          </button>
        </LinkRoutes>
        <LinkRoutes
          key="1"
          route="/party/"
        >
          <button>
            <IosAdd
              beat={false}
              color="#7A7A7A"
              fontSize="60px"
              rotate={false}
              shake={false}
              style={Object {}}
            />
            <h3>
              Create your own
            </h3>
          </button>
        </LinkRoutes>
      </div>
    </div>
  </div>
</main>
`;
